gpt4all-chat/modellist.cpp

```diff
@@ -1204,9 +1241,14 @@ void ModelList::updateModelsFromDirectory()
         }
     };
 
+
+    updateOldRemoteModels(exePath);
     processDirectory(exePath);
-    if (localPath != exePath)
+    if (localPath != exePath) {
+        updateOldRemoteModels(localPath);
         processDirectory(localPath);
+    }
+
 }
```

Comment:
I don't see a need for this blank line:
```suggestion
    }
}
```